server/license.go

```diff
@@ -10,5 +10,7 @@ import (
 )
 
 func isLicensed(_ *model.License, api *pluginapi.Wrapper) bool {
-	return originalapi.IsE20LicensedOrDevelopment(api.Configuration.GetConfig(), api.System.GetLicense())
+	license := api.System.GetLicense()
+	config := api.Configuration.GetConfig()
+	return originalapi.IsE20LicensedOrDevelopment(config, license) || (!originalapi.IsConfiguredForDevelopment(config) && license != nil && license.SkuShortName != "entry")
```

Comment:
Shouldn't this be an &&? `IsE20LicensedOrDevelopment` will return true for Entry as it's doing the minimum Enterprise check

Reply:
I made an adjustment to the condition in the hopes of making it more readable. The "OrDevelopment" part of all the license checks in the plugin API make things a little weird

It will return true if
The system is in development mode
The system is not in development AND it has a non-entry valid license